web/concrete/src/Application/Application.php

```diff
@@ -130,6 +130,45 @@ public function clearCaches()
         OpCache::clear();
     }
 
+   /**
+     * Utility method for clearing page caches and block caches that has btCacheBlockOutputClearOnEvents option enabled
+     */
+    public function partialClearCaches()
+    {
+        \Events::dispatch('on_partial_cache_flush');
```

Comment:
We have that with `\Database` and `\Events`, these convenience layers are expensive to use however and so we should take care in the core.